### DISCUSSION TITLE
Community Meeting timing/day

### DISCUSSION THREAD

**Discussion by stockholmux** [bottlerocket-os/bottlerocket#2372] [General]
How does 8:30am (Pacific) on Wednesdays (every two weeks) work for everyone? Opinions welcome in this thread!

**Comment by misterek** [comment]
Works great for me.

**Comment by stockholmux** [comment]
Hearing none - our next meeting scheduled: 
https://www.meetup.com/bottlerocket-community/events/288085016/

See you in a couple of weeks!
